tests/settings.py

```diff
@@ -33,6 +36,16 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 )
 
+if VERSION < (1, 10, 0):
+    MIDDLEWARE_CLASSES = (
```

Comment:
Could this just be `MIDDLEWARE_CLASSES = MIDDLEWARE`?

Reply:
yes! will update. Working on other more complicated django-2.0 support